src/components/LogoFooter.tsx

```diff
@@ -0,0 +1,41 @@
+import Button from 'components/Button'
+import Logo from 'components/Logo'
+import Twitter from 'icons/Twitter'
+import classnames, {
+  alignItems,
+  display,
+  flexDirection,
+  justifyContent,
+  space,
+} from 'classnames/tailwind'
+
+const container = classnames(
```

Comment:
Maybe let's name it `wrapper` or `box`? You already have a container here

Reply:
Did it. Thx